tools/ci_build/github/azure-pipelines/build-perf-test-binaries-pipeline.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+- name: BuildAndroidBinaries
+  type: boolean
+  default: true
+
+jobs:
+- ${{ if parameters.BuildAndroidBinaries }}:
+  # build binaries for Android
+  - template: templates/android-java-api-aar.yml
+    parameters:
+      buildConfig: 'Release'
+      buildSettings: '$(Build.SourcesDirectory)/tools/ci_build/github/android/default_full_aar_build_settings.json'
+      artifactName: 'onnxruntime-android-full-aar'
+      job_name_suffix: 'Full'
+      publish_executables: '1'
+      pool_name: 'Linux-CPU-2019'
```

Comment:
If it runs daily, I suggest putting it in https://aiinfra.visualstudio.com/. Most of our daily pipelines are there.

Reply:
moved to aiinfra

Reply:
actually, maybe it should run on every master commit. would aiinfra still be the right place?